qt/PrefsDialog.cc

```diff
@@ -578,9 +578,16 @@ void PrefsDialog::initSeedingTab()
 
 void PrefsDialog::onQueueStalledMinutesChanged()
 {
-    //: Spin box suffix, "Download is inactive if data sharing stopped: [ 5 minutes ago ]" (includes leading space after the number, if needed)
-    QString const units_suffix = tr(" minute(s) ago", nullptr, ui_.queueStalledMinutesSpin->value());
+    //: Spin box format, "Download is inactive if data sharing stopped: [ 5 minutes ago ]" (includes leading space after the number, if needed)
+    QString const units_format = tr("%1 minute(s) ago", nullptr, ui_.queueStalledMinutesSpin->value());
 
+    QString const units_prefix = units_format.section(QStringLiteral("%1"), -2, -2);
+    QString const units_suffix = units_format.section(QStringLiteral("%1"), -1);
```

Comment:
Slightly ineffective due to double search, and a bit weird to count from the end (why?). Maybe
```suggestion
    auto const placeholder = QStringLiteral("%1");
    auto const placeholder_pos = units_format.indexOf(placeholder);
    if (placeholder_pos == -1)
    {
        return;
    }

    auto const units_prefix = units_format.left(placeholder_pos);
    auto const units_suffix = units_format.mid(placeholder_pos + placeholder.size());
```

Note how we don't care if translators broke the string and didn't include the placeholder — not our problem.

Reply:
I counted from the back, because that wouldn't break already translated strings. Not including the placeholder would default to just suffix.

Reply:
Nah, doesn't work that way. It's a new phrase and it'll get its own new translation in every language. Omitting the placeholder is not an option.